modules/articles/client/config/articles.client.routes.js

```diff
@@ -12,22 +12,52 @@ angular.module('articles').config(['$stateProvider',
       })
       .state('articles.list', {
         url: '',
-        templateUrl: 'modules/articles/client/views/list-articles.client.view.html'
+        templateUrl: 'modules/articles/client/views/list-articles.client.view.html',
+        controller: 'ArticlesListController'
       })
       .state('articles.create', {
         url: '/create',
         templateUrl: 'modules/articles/client/views/create-article.client.view.html',
+        controller: 'ArticlesController',
         data: {
           roles: ['user', 'admin']
+        },
+        resolve: {
+          article: function () {
+            return { };
```

Comment:
Because article controller is now expecting to resolve an article.  Its either send a blank object or split it to another controller